graphql_compiler/tests/test_compiler.py

```diff
@@ -5722,7 +5754,7 @@ def test_output_count_in_fold_scope(self):
                 "Animal_1".name AS name,
                 folded_subquery_1.fold_output__x_count AS number_of_children
             FROM schema_1."Animal" AS "Animal_1"
-            LEFT OUTER JOIN (
+            JOIN (
```

Comment:
All of these changes are because #666 switched to JOINs instead of LEFT OUTER JOINs but these tests were made in #621